docs/tab-widgets/distributed-trace-receive.asciidoc

```diff
@@ -0,0 +1,163 @@
+// tag::go[]
+
+// Need help with this example
+
+1. Get the incoming TraceContext.
+
+2. Start a new transaction or span as a child of the incoming transaction with
+{apm-go-ref}/api.html#tracer-api-start-transaction-options[`StartTransactionOptions`] or
+{apm-go-ref}/api.html#transaction-start-span-options[`StartSpanOptions`]
+
+Example starting a new transaction:
+
+[source,go]
+----
+// Receive incoming TraceContext
```

Comment:
```suggestion
// Receive incoming TraceContext
traceContext, _ := apmhttp.ParseTraceparentHeader(r.Header.Get("Traceparent"))
traceContext.State, _ = apmhttp.ParseTracestateHeader(r.Header["Tracestate"]...)
```

Reply:
In this sample code, `r` refers to a [net/http.Request](https://golang.org/pkg/net/http/#Request) being handled by a web service.